Adds attributes for data views

szabosteve opened the pull request:
## Overview

This PR introduces two attributes to the list:

* `data-source-caps`. Resolves to `Data View`.
* `data-source-cap`. Resolves to `Data view`.

gtback reviewed (approved):
LGTM! But why are there so many versions of this variable?

szabosteve commented:
> LGTM! But why are there so many versions of this variable?

Some of the versions are not correct. The system doesn't respect whether the attribute itself contains capitalization or not. I'll delete the incorrect ones once we added the correct versions and changed the docs to use the new attributes.
